plugin/src/shared/generators/navigation-utils/navigation-lib-files/shared/utils/navigation/src/lib/config.ts.template

```diff
@@ -1,5 +1,9 @@
 export const navigationConfig = {
   routes: {
-    // Write app routes here
+    // TODO: Write app routes here. Example:
+    public: {
+      signIn: '/sign-in',
```

Comment:
Please, add template with this page or change the redirect in the proxy. After generating an app 404 is always open